addons/payment_stripe/static/src/js/express_checkout_form.js

```diff
@@ -179,6 +189,9 @@ paymentExpressCheckoutForm.include({
                         },
                     },
                 );
+                const adjustedAmountWithFiscalPosition = await this.rpc(
+                    this.paymentContext['shippingAddressUpdateRoute'] + "/compute_taxes",
```

Comment:
```suggestion
                    this.paymentContext['shippingAddressUpdateRoute'] + '/compute_taxes',
```
It avoids adding a new route in stable but maybe change it by adding one in master.